### DISCUSSION TITLE
Help me with my naming convention

### DISCUSSION THREAD

**Discussion by TheMissingPort** [Serene-Arc/bulk-downloader-for-reddit#931] [Q&A]
bdfr download  ./REDDITUSER user Reddituser --submitted --config ./opt/homebrew/lib/python3.11/site-packages/bdfr/my_config.cfg 

I want it to download into the REDDITUSER name folder. 

Also, how do I make it name the files as {DATE}

Thanks

**Comment by Serene-Arc** [comment]
Is there a reason you only want one file per day?

**Reply by TheMissingPort** [reply]
Oops sorry for the oversight

**Reply by Serene-Arc** [reply]
What did you want to accomplish?

**Reply by TheMissingPort** [reply]
What it says in my original post

**Reply by Serene-Arc** [reply]
So you did want a single file per day?

**Reply by TheMissingPort** [reply]
where in my post did I say that

**Reply by Fakeaccount12312** [reply]
Multiple files can't have the same name. If you want only the date as filename, you can only create one File per day/date. I assume you wanted something different?

**Reply by Fakeaccount12312** [reply]
Otherwise everything you need to know is in the description of the options --file-scheme and --folder-scheme, there are also examples, its very descriptive. Try and read/use these and if you still have issues come back again, but they should work.
